ergo/distributions/truncated.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass
+from .distribution import Distribution
+import jax.numpy as np
+
+
+def truncate(underlying_dist_class: Distribution, floor: float, ceiling: float):
```

Comment:
Good point. @stuhlmueller the issue was that I couldn't easily get `from_params` to work if the parameters to `truncate` were included as fields on the `TruncatedDist` class

Just communicating that here as a comment for now because I'm hoping we'll resolve it before merging this code